stable/meteor/values.yaml

```diff
@@ -9,6 +9,7 @@ labels: {"application":"meteor"}
 
 config: {}
 secretConfig: {}
+ssl_secrets: []
```

Comment:
This should be optional with enabled flag

Reply:
kept the default value as empty, it would not make any change if we do not pass in the value.